recipes/drop/meta.yaml

```
@@ -30,7 +30,7 @@ requirements:
 
     # snakemake/wbuild
     - snakemake >=5.5.2
-    - wbuild =1.7.0
+    - wbuild =1.7.1
```

We should not lock the version if its not needed.

```suggestion
    - wbuild >=1.7.1
```